frontend/src/App.css

```diff
@@ -36,3 +36,23 @@
     transform: rotate(360deg);
   }
 }
+
+.Games-grid {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  gap: 16px;
+}
+
+.Grid-item {
+  padding: 20px;
+  text-align: center;
+  background-color: #61dbfb;
+  border-radius: 8px;
+  text-decoration: none;
```

Comment:
ar reikia paminėti, kad text decoration none ? Man atrodo default turėtų būt none. Jeigu netingi pabandyk patikrint ar taspats be jo. Visada bandom rašyt kuo mažiau nereikalingo kodo, kad codebase netaptu bloated.

Reply:
patikrinau, reikia, underlined kaip hyperlink'as palieka